core/src/main/res/values/environment_config.xml

```diff
@@ -40,8 +40,8 @@
 
     <string name="alarmErrorsServiceBaseUrl">https://oncall.keyapp.org/</string>
 
-    <string name="strigaProxyServiceBaseUrl">https://payment-rust.keyapp.org/striga/api/</string>
-    <string name="strigaProxyServiceProdBaseUrl">https://payment.key.app/striga/api/</string>
+    <string name="strigaProxyServiceBaseUrl">https://payment-rust.keyapp.org/</string>
```

Comment:
do you need to add prod strigaProxyServiceBaseUrl in release strings.xml?

Reply:
it's not used now, just historically they were there together, dev and prod urls, but yes, you're right, it's better to move prod url to the `release` strings to avoid doing this later